### PR TITLE
Add info about contention profiling to profiling.md

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: docs/devel/profiling.md [hunk 1]
```diff
@@ -23,8 +23,12 @@ Even when running profiler I found not really straightforward to use 'go tool pp
...
-or analogous one for you Cloud provider. Afterwards you can e.g. run 
+or analogous one for you Cloud provider. Afterwards you can e.g. run
 ```
 go tool pprof http://localhost:<local_port>/debug/pprof/profile
 ```
 to get 30 sec. CPU profile.
+
+## Contention profiling
+
+To enable contetion profiling you need to add line ```rt.SetBlockProfileRate(1)``` to ones added before (```rt``` stands for ```runtime``` in ```master.go```). This enables 'debug/pprof/block' subpage, which can be used as an input go to ```go tool pprof```.
```
Comment (yujuhong) [inline docs/devel/profiling.md]: s/go to/to

Comment (yujuhong) [inline docs/devel/profiling.md]: Might be useful to clarify "to ones added before".

**Comment by gmarek** [comment]
Done.

**Comment by yujuhong** [comment]
LGTM

**Comment by yujuhong** [comment]
I don't think the test failures were relevant, but I kicked them off again just to be cautious.

**Pull request merged**